### PR TITLE
Setup docker+compose for environment configuration

### PR DESCRIPTION
**Pull request by MarkRoddy**

### CODE REVIEW

File: python-flask-todo-list/app/__init__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ def create_app(test_config=None):
     app.config.from_mapping(
         SECRET_KEY = 'dev', # Replace in production!
         DATABASE = os.path.join(app.instance_path, 'todo.sqlite'),
-        NILE = 'http://localhost:8080',
+        NILE = os.environ.get('NILE', 'http://localhost:8080'),
```
Comment (gwenshap) [inline python-flask-todo-list/app/__init__.py:11]: Nice :) Flask didn't have support for env variables, and I didn't think of doing the simplest thing....

**Review by gwenshap — APPROVED** [review]
LGTM. Thank you so much @MarkRoddy for being the first contributor 🥇 
I tested the container on my local machine and it worked perfectly. We appreciate your support.

**Comment by gwenshap** [comment]
LOL. I forgot that @MarkRoddy can't merge it :) Merging now.

**Comment by MarkRoddy** [comment]
> I forgot that @MarkRoddy can't merge it 
For the best for everyone involved 😂

**Pull request merged**